haskell-xmpp.cabal

```
@@ -8,7 +8,7 @@ Homepage: https://github.com/riskbook/haskell-xmpp
 Category: Network
 Copyright: (c) 2005-2011 Dmitry Astapov, k.pierre
 Stability: Stable
-Cabal-version: >=2.00
+Cabal-version: >=2
```

I believe the syntax is just:
```suggestion
cabal-version: 2.0
```
Also could you please make it the first line in this file? It is important to have it as the first place as the further parsing of the cabal syntax depends on the version.